src/main/scala/com/gu/support/workers/model/ErrorState.scala

```diff
@@ -0,0 +1,3 @@
+package com.gu.support.workers.model
+
+case class ErrorState(Error: String, Cause: String = "")
```

Comment:
Yup so do I but unfortunately they are passed in by AWS which specifies upper case
http://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-errors.html
 
I could use a decapitalising encode in support-workers I guess, do you think that's worth doing?